ts/features/payments/receipts/screens/ReceiptListScreen.tsx

```diff
@@ -163,14 +163,27 @@ const ReceiptListScreen = () => {
     </>
   );
 
+  const emptyProps: {
+    title: string;
+    subtitle?: string;
+    pictogram: IOPictograms;
+  } =
```

Comment:
I noticed you’ve defined a custom type for emptyProps. Since these properties are closely tied to `OperationResultScreenContent`, I suggest creating a derived type using `Pick`. This will make the code more maintainable and consistent with the component’s actual props. Example:

```typescript
type OperationResultEmptyProps = Pick<
  typeof OperationResultScreenContent.propTypes,
  "title" | "subtitle" | "pictogram"
>;
```
This also helps avoid duplicating definitions if the props change in the future. Let me know what do you think about.

Reply:
What do you think to use `OperationResultScreenContentProps` type instead  `typeof OperationResultScreenContent.propTypes`?

Reply:
Yes, it makes more sense. I saw that the type `OperationResultScreenContentProps` was not exported. So in this case it's better to export it and use it 👍

Reply:
You can find it here d9cbd7f98a6102496de3ca3bb36873df8d392a79